syntax = "proto3";

package journey11;

message HelloWorld {
    string person_name = 1; // Name to say hello to.
    map<string, int32> tasks = 2;
}

